libs/wingc/src/comp_ctx.rs

```diff
@@ -105,8 +102,9 @@ pub fn set_custom_panic_hook() {
 
 		report_diagnostic(Diagnostic {
 			message: format!(
-				"Compiler bug ({}) during {}, please report at https://www.winglang.io/contributing/start-here/bugs",
-				pi,
+				"Compiler bug ('{}' at {}) during {}, please report at https://www.winglang.io/contributing/start-here/bugs",
+				pi.payload().downcast_ref::<&str>().unwrap_or(&""),
+				pi.location().unwrap_or(&std::panic::Location::caller()),
```

Comment:
The new Display for panicinfo changed in a recent rust version. IMO it didn't look good here, so here's a new look.